packages/app/src/elements/import-dialog.ts

```diff
@@ -20,15 +26,66 @@ export class ImportDialog extends Dialog<File, void> {
     @state()
     private _items: VaultItem[] = [];
 
+    @state()
+    private _itemColumns: imp.ImportCSVColumn[] = [];
+
+    @query("#csvHasDataOnFirstRowCheckbox")
+    private _csvHasDataOnFirstRowCheckbox: HTMLInputElement;
+
     @query("#formatSelect")
     private _formatSelect: Select<string>;
 
     @query("#vaultSelect")
     private _vaultSelect: Select<Vault>;
 
+    @query("#nameColumnSelect")
+    private _nameColumnSelect: Select<number>;
+
+    @query("#tagsColumnSelect")
+    private _tagsColumnSelect: Select<number>;
+
+    @queryAll("pl-select.field-type-select")
+    private _fieldTypeSelects: Select<FieldType>[];
+
+    @queryAll("pl-input.field-name-input")
+    private _fieldNameInputs: Input[];
+
+    static styles = [
+        ...Dialog.styles,
+        css`
+            :host {
+                --pl-dialog-max-width: 40em;
+            }
+        `,
+    ];
+
+    // TODO: Fix dialog reset after closed (when re-opened)
+
     renderContent() {
+        // TODO: Fix wrong first selected value for selects
```

Comment:
@MaKleSoft also, do you have any idea on what I should look at? These two `pl-select`s just start with the first option, no matter what I set below. I've tried setting them with `.value` like the other `pl-select`s but it didn't work.

I've tried a few things around using a `string` and a `number`, in case that was causing issues.

Reply:
@MaKleSoft not sure if you missed this one, but I'm more at a loss for it right now.